### PR TITLE
Update setting_up_git.md

### PR DESCRIPTION
**Pull request by G-O-J-O-U-S-A-T-O-R-U**
## Because

The normal command prompt does not work for legacy system users and thus this helps.

## Issue

Closes #XXXXX
Related to ---
## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by G-O-J-O-U-S-A-T-O-R-U** [comment]
The command prompt to copy SSH key will not work properly for legacy system users since its beneficial to add the command prompt that will work for legacy system users.

**Comment by ManonLef** [comment]
I removed the `Related to:` issue from your post. It does not seem related to that particular PR

File: foundations/installations/setting_up_git.md [hunk 1]
```diff
@@ -197,7 +197,14 @@ Now you need to copy your public SSH key. To do this, we're going to use a comma
 cat ~/.ssh/id_ed25519.pub
 ~~~
 
+**NOTE:** For a legacy system use the following command propmt in the git bash to copy your public SSH key.
```
Comment (CBID2) [inline foundations/installations/setting_up_git.md:200]:
```suggestion
**NOTE:** For a legacy system, use the following command prompt in the git bash to copy your public SSH key.
```

File: foundations/installations/setting_up_git.md [hunk 2]
```diff
@@ -197,7 +197,14 @@ Now you need to copy your public SSH key. To do this, we're going to use a comma
...
+
+~~~bash
+cat ~/.ssh/id_rsa.pub
+~~~
+
 Highlight and copy the output, which starts with `ssh-ed25519` and ends with your email address.
+Highlight and copy the output, which starts with `ssh-rsa` and ends with your email address for legacy users.
 
 Now, go back to GitHub in your browser window and paste the key you copied into the key field. Keep the key type as `Authentication Key` and then, click `Add SSH key`. You're done! You've successfully added your SSH key!
```
Comment (CBID2) [inline foundations/installations/setting_up_git.md:209]: [Proofreading/nitpick] The ending of this paragraph seems repetitive, so check out my suggestion below to see how you can improve it. 
```suggestion
Now, go back to GitHub in your browser window and paste the key you copied into the key field. Keep the key type as `Authentication Key` and then, click `Add SSH key`. Congrats, you've successfully added your SSH key!
```

**Review by CBID2 — COMMENTED** [review]
Hi @G-O-J-O-U-S-A-T-O-R-U! :) I found some proofreading errors in your pull request. If you find my suggestions helpful, check out [this article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) to learn how to apply them to your PR. Other than that, you're off to a great start! :)

**Comment by xandora** [comment]
@CBID2 I suggest waiting until the requested reviewers have had a chance to look at PRs before offering your own suggestions. As it stands, this PR isn't going to be included, so your time has been wasted here.

@G-O-J-O-U-S-A-T-O-R-U What do you define as a legacy system user? Because if they're not following along with our curriculum, it's out of support anyway. All LTS versions of Ubuntu shouldn't need to use this at all. Additionally, your mention of `git bash` in the PR makes me believe this came from a Windows system, which is definitely not in our current support platform and not something we'd include here anyway.

If you can show how this might impact a user following our curriculum recommendations, we can reconsider opening this PR.

File: foundations/installations/setting_up_git.md (hunk 2 above)
Comment (G-O-J-O-U-S-A-T-O-R-U) [reply foundations/installations/setting_up_git.md:209]: Thank you for helping me with the pull request, since I'm new here, your help means a lot.

**Comment by G-O-J-O-U-S-A-T-O-R-U** [comment]
> 

It might not affect the users that has Ubuntu, but mostly students who learn from various parts seem to have windows with them as default and since the obtaining of the public key would be a little different for them it would be really helpful for users who are learning from windows. Since this is the beginning for a new learner to set up git, and most students from various countries have windows it makes a little harder for them to set up git or obtain the SSH key with these command prompts.  A beginner won't be aware of his system completely and upon using the CMD for ubuntu will confuse them and will be blocking their progress to the next stage. The rest of the CMD works fine in the Windows, except for the CMD required for obtaining and  to copy the SSH key. Hope this explanation would be really helpful in reviewing the issue once again. :)

**Pull request closed without merging**